package.json

```diff
@@ -37,7 +37,8 @@
     "pull-stream": "^3.7.0",
     "sodium-native": "^3.4.1",
     "ssb-bfe": "^3.7.0",
-    "ssb-crut": "^4.6.2",
+    "ssb-box2": "^7.1.0",
```

Comment:
:fire: you can't use this without meta-feeds

https://github.com/ssbc/ssb-box2/blob/master/monitor.js#L40

I don't want to add it, because too much I don't trust and don't want to maintain by myself.

Reply:
oooh, `box2.legacyMode` huh nice I didn't see that

Reply:
resolved?